<launch>
    <arg name="id" /> 
    <arg name="teammates_id" />
    <arg name="is_leader" />

	<node pkg="strategy" type="main_robot.py" name="main_robot"  output="screen" respawn="false" required="true">   
    <param name="id" value="$(arg id)" />
    <param name="teammates_id" value="$(arg teammates_id)"/>
    <param name="is_leader" value="$(arg is_leader)"/>
    </node>

</launch>
